build_tools/travis/install.sh

```diff
@@ -112,6 +115,14 @@ else
     python --version
     python -c "import numpy; print('numpy %s' % numpy.__version__)"
     python -c "import scipy; print('scipy %s' % scipy.__version__)"
+    python -c "\
+try:
```

Comment:
from what I understand from the change above, pandas is installed in all the cases, so I don't think the try/except is needed.